### PR TITLE
New code/MI-10-MEAN/Pratiksha/ Changes Done in javascript assignment-4

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

**Review by poonam-khode — CHANGES_REQUESTED** [review]
proper structure and remove extra space

**Review by poonam-khode — APPROVED** [review]

File: Pratiksha/Javascript/Assignment4/Q1.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q1.html]: remove extra line and proper structure

File: Pratiksha/Javascript/Assignment4/Q1.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Armstrong</title>
+</head>
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q1.html]: remove extra line

File: Pratiksha/Javascript/Assignment4/Q1.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+<body>
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q1.html]: same as 2 Comment

File: Pratiksha/Javascript/Assignment4/Q1.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  Enter any Number: <input id="val1">
+  <button onclick="armstrg()">Check</button></br></br>
+  <p id="demo"></p>
+  <script>
+    function armstrg() {
+      var arm = 0, a, b, c, d, num;
+      num = Number(document.getElementById("val1").value);
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q1.html]: same as

File: Pratiksha/Javascript/Assignment4/Q1.html [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      }
+      if (arm == num) {
+        alert("Armstrong number");
+      }
+      else {
+        alert("Not Armstrong number");
+      }
+    }
+  </script>
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q1.html]: same as

File: Pratiksha/Javascript/Assignment4/Q1.html [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+</body>
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q1.html]: same as 1 comment

File: Pratiksha/Javascript/Assignment4/Q3.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q3.html:3]: proper structure

File: Pratiksha/Javascript/Assignment4/Q3.html [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+            }
+            sum = sum + p;
+            num = Math.floor(num/10);
+        }
+        if (sum == save_num) {
+            document.getElementById("demo").innerText =save_num + " is a Krishnamurthy Number";
+        } else {
+            document.getElementById("demo").innerText =save_num + "is not a KrishnamurthyNumber";
+        }
+    }</script>
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q3.html]: what need double close Script

File: Pratiksha/Javascript/Assignment4/Q4.html [hunk 9]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q4.html:10]: remove extra line and proper structure please

File: Pratiksha/Javascript/Assignment4/Q4.html [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+             document.getElementById("demo").innerHTML = "Vowel "
+            break;
+
+            default:
+             document.getElementById("demo").innerHTML = ""+x+" Not Vowel"
+             
+        }
+        event.preventDefault();
+}
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q4.html:57]: what is this Pratikaha please remove all extra line

File: Pratiksha/Javascript/Assignment4/Q5.html [hunk 11]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q5.html:3]: proper structure

File: Pratiksha/Javascript/Assignment4/Q5.html [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+    <title>Document</title>
+</head>
+<body>
+    <h1>UpperCase And LowerCase </h1>
+    <form>
+        ENter String:
+        <input type="text" id="val1">
+        <button onclick="upper()">UpperCase</button>
+        <button onclick="lower()">LowerCase</button>
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q5.html:16]: remove extra line

File: Pratiksha/Javascript/Assignment4/Q5.html [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+            var string = str.toUpperCase();
+            document.getElementById("demo").innerText = `${string}`;
+        }
+
+        function lower() {
+            var str = 'geeksforgeeks';
+            var string = str.toUpperCase();
+            document.getElementById("demo").innerText = `${string}`
+        }
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment4/Q5.html:31]: remove extra line

File: Pratiksha/Javascript/Assignment5/Q1.html [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+        <input type="text" id="number">
+        <br>
+        <button type="submit" onclick="sumFirstAndLastEqualMiddleDigit()">Sum</button>
+    </form>
+    <p id="sum"></p>
+    <p id="result"></p>
+    <script>
+        function sumFirstAndLastEqualMiddleDigit() {
+    event.preventDefault();
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment5/Q1.html:22]: remove extra line

File: Pratiksha/Javascript/Assignment5/Q1.html [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+    let number = parseInt(document.getElementById('number').value);
+    let result = document.getElementById('result');
+    let sum = document.getElementById('sum');
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment5/Q1.html:26]: remove extra line

File: Pratiksha/Javascript/Assignment5/Q4.html [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+        result.innerHTML = `Sum: ${listOfEven.join("+")} = ${sumOfEven} <br> 
+                            Prodcut: ${listOfOdd.join("*")} = ${productOfOdd}`;
+    }
+
+    else {
+        result.innerHTML = "Invalid Number!! Please enter a 6 digit number."
+    }
+
+} 
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment5/Q4.html:55]: remove extra line

File: Pratiksha/Javascript/Assignment5/Q5.html [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+    let result = document.getElementById('result');
+    let counter = 1;
+
+    for (let row=1; row <= number; row++) {
+
+        for(let column = 1; column <= row; column++) {
+            result.innerHTML += ` ${counter} `
+            counter++;
+        }
+
```
Comment (VIP5614) [inline Pratiksha/Javascript/Assignment5/Q5.html:32]: remove extra line

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+              </tr>
+            </tbody>
+          </table>
+    </div>
+
+
+
+
+
+
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html:59]: remove extra lines

File: Pratiksha/Bootstrap/Assignment5/Assignment5.html [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0"
+        crossorigin="anonymous"></script>
+</body>
+</html>
```
Comment (KomalShinde813) [inline Pratiksha/Bootstrap/Assignment5/Assignment5.html:72]: add extra line at the end of file

File: Pratiksha/Javascript/Assignment2/Q1.html [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+      Number4:
+      <input type="text" id="val3" /><br /><br />
+      Number5:
+      <input type="text" id="val4" /><br /><br />
+      <button type="button" class="ml-4 btn btn-primary center">
+        Largest among 5
+      </button>
+    </form>
+    <p id="largest"></p>
+    <script></script>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment2/Q1.html]: incomplete program don't push incomplete work on git

File: Pratiksha/Javascript/Assignment2/Q2.html [hunk 21]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment2/Q2.html]: change title in every assignment

File: Pratiksha/Javascript/Assignment2/Q2.html [hunk 22]
```diff
@@ -0,0 +1,73 @@
...
+        type="button"
+        class="ml-4 btn btn-primary center"
+      >
+        Grade
+      </button>
+    </form>
+    <p id="demo"></p>
+    <p id="demo1"></p>
+    <script>
+      var c;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment2/Q2.html:46]: use proper variable name don't use a, b, c, d, x, y, z

File: Pratiksha/Javascript/Assignment2/Q2.html [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+        y = parseInt(document.getElementById("val2").value);
+        z = parseInt(document.getElementById("val3").value);
+        a = parseInt(document.getElementById("val4").value);
+        b = parseInt(document.getElementById("val5").value);
+        c = ((x + y + z + a + b) / 500) * 100;
+
+        document.getElementById("demo").innerText = `Percentage ${c}`;
+      }
+      function grade() {
+        if (c >= 75 && c <= 100) {
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment2/Q2.html:58]: no need of curly brackets for single if condition

File: Pratiksha/Javascript/Assignment3/Q1.html [hunk 24]
```diff
@@ -0,0 +1,30 @@
...
+      </button>
+    </form>
+    <p id="demo"></p>
+    <script>
+      n = document.getElementById("val1").value;
+      function leap() {
+        if (n % 4 == 0) {
+          document.getElementById("demo").innerText = `Leap year`;
+        } else {
+          document.getElementById("demo").innerText = `  Not Leap year`;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q1.html:24]: remove extra space

File: Pratiksha/Javascript/Assignment3/Q1.html [hunk 25]
```diff
@@ -0,0 +1,30 @@
...
+        }
+        event.preventDefault();
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q1.html:26]: what is the use of this line?

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 26]
```diff
@@ -0,0 +1,52 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prime</title>
+</head>
+<body>
+   
+    
+
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:12]: remove extra line spaces

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 27]
```diff
@@ -0,0 +1,52 @@
...
+            
+              
+            <h4>check number is prime or not</h4>
+              
+        
+              
+            <form name="myform">
+                Enter the number:
+                <input type="text" name=n value="">
+
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:22]: remove extra lines from everywhere

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 28]
```diff
@@ -0,0 +1,52 @@
...
+              
+        
+              
+            <form name="myform">
+                Enter the number:
+                <input type="text" name=n value="">
+                  
+                <br><br>
+                  
+                <input type="button" value="Check" onClick="p()">
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:25]: change function name

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 29]
```diff
@@ -0,0 +1,52 @@
...
+                    flag = false;
+                    break;
+                }
+                  
+                // Check and display alert message
+            if (flag == true)
+                document.getElementById("demo").innerText= `prime`
+            else
+            document.getElementById("demo").innerText= ` Not prime`        }
+    </script>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:49]: check indentation properly

File: Pratiksha/Javascript/Assignment3/Q2 .html [hunk 30]
```diff
@@ -0,0 +1,52 @@
...
+    </html>
+</body>
+</html>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q2 .html:52]: extra line required at the end of file

File: Pratiksha/Javascript/Assignment3/Q3.html [hunk 31]
```diff
@@ -0,0 +1,28 @@
...
+  </head>
+  <body>
+    <form>
+      Enter year:
+      <input type="text" id="val1" /><br /><br />
+      <button class="button" onclick="evod()">Even or ODd</button>
+    </form>
+    <p id="demo"></p>
+    <script>
+      n = document.getElementById("val1").value;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q3.html]: change variable name

File: Pratiksha/Javascript/Assignment3/Q3.html [hunk 32]
```diff
@@ -0,0 +1,28 @@
...
+      function evod() {
+        if (n / 2 == 0) {
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q3.html]: no need of curly brackets for single if condition

File: Pratiksha/Javascript/Assignment3/Q_2.html [hunk 33]
```diff
@@ -0,0 +1,44 @@
...
+    <form name="myform">
+        Enter the number:
+        <input type="text" name=n value="">
+        <br><br>
+        <input type="button" value="Check" onClick="p()">
+        <br>
+    </form>
+    <p id="demo"></p>
+    <script>
+        function p() {
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q_2.html:22]: change function name

File: Pratiksha/Javascript/Assignment3/Q_2.html [hunk 34]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment3/Q_2.html:1]: duplicate program

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 35]
```diff
@@ -0,0 +1,36 @@
...
+  <body>
+    <form>
+      <input type="text" id="first" placeholder="Enter a number" />
+      <button onclick="automorphic()">Submit</button>
+    </form>
+    <p id="demo"></p>
+    <p id="demo1"></p>
+    <script>
+      function automorphic(){
+        var n,m;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html]: space required after ,

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 36]
```diff
@@ -0,0 +1,36 @@
...
+  </head>
+  <body>
+    <form>
+      <input type="text" id="first" placeholder="Enter a number" />
+      <button onclick="automorphic()">Submit</button>
+    </form>
+    <p id="demo"></p>
+    <p id="demo1"></p>
+    <script>
+      function automorphic(){
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html]: space required before {

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 37]
```diff
@@ -0,0 +1,36 @@
...
+        var n,m;
+        n = parseInt(document.getElementById ("first").value);
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html]: remove  extra space

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 38]
```diff
@@ -0,0 +1,36 @@
...
+        m = n*n;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html]: space required in operators

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 39]
```diff
@@ -0,0 +1,36 @@
...
+        document.getElementById ('demo').innerHTML ="Square of a number is:" +m;
+        if(n%10 != m%10)
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html]: space required in operators and after if

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 40]
```diff
@@ -0,0 +1,36 @@
...
+        {
+        document.getElementById('demo1').innerHTML = n +" is not an atomorphic number";
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html]: check indentation

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 41]
```diff
@@ -0,0 +1,36 @@
...
+        n = n/10;
+        m = m/10;
+        }
+        else
+        {
+        document.getElementById('demo1').innerHTML = n +" is an automorphic number";
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html]: check indentation

File: Pratiksha/Javascript/Assignment4/Q2.html [hunk 42]
```diff
@@ -0,0 +1,36 @@
...
+        }
+        event.preventDefault();
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q2.html]: what is the use of this line here?

File: Pratiksha/Javascript/Assignment4/Q3.html [hunk 43]
```diff
@@ -0,0 +1,38 @@
...
+    <title>Document</title>
+</head>
+<body>
+    
+    Enter Number :- <input type="text" id="val"> <br><br>
+    <button onclick="KrishnamurthyNumber()">KrishnamurthyNumber</button>
+    <br><br>
+    <div id="demo"></div>
+    <script>function KrishnamurthyNumber() {
+        var num,i,p,r,sum=0,save_num;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q3.html]: space required after ,

File: Pratiksha/Javascript/Assignment4/Q3.html [hunk 44]
```diff
@@ -0,0 +1,38 @@
...
+        num = parseInt(document.getElementById("val").value);
+        save_num = num;
+        while (num) {
+            i=1;
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q3.html]: space required before and after operator make this change everywhere

File: Pratiksha/Javascript/Assignment4/Q3.html [hunk 45]
```diff
@@ -0,0 +1,38 @@
...
+        }
+        if (sum == save_num) {
+            document.getElementById("demo").innerText =save_num + " is a Krishnamurthy Number";
+        } else {
+            document.getElementById("demo").innerText =save_num + "is not a KrishnamurthyNumber";
+        }
+    }</script>
+</script>
+</body>
+</html>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q3.html:39]: extra line required at the end of file

File: Pratiksha/Javascript/Assignment4/Q4.html [hunk 46]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q4.html:3]: check indentation <head> is under <html>

File: Pratiksha/Javascript/Assignment4/Q4.html [hunk 47]
```diff
@@ -0,0 +1,71 @@
...
+    <form>
+        Enter a character:
+        <input type="text" name="char" id="ch">
+        <button onclick="check()">Check</button>
+    </form>
+    <p id="demo"></p>
+    <script> 
+        function check(){
+    x = document.getElementById("ch").value;
+        switch(x){
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q4.html:20]: space required

File: Pratiksha/Javascript/Assignment4/Q4.html [hunk 48]
```diff
@@ -0,0 +1,71 @@
...
+            case "A":
```
Comment (KomalShinde813) [inline Pratiksha/Javascript/Assignment4/Q4.html:21]: add a here in || condition and remove duplicate code

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by sanketmod — APPROVED** [review]

File: Pratiksha/Javascript/Assignment3/Q1.html (hunk 25 above)
Comment (pratikshaghule) [reply Pratiksha/Javascript/Assignment3/Q1.html:26]: to hold on the screen

File: Pratiksha/Javascript/Assignment3/Q3.html (hunk 31 above)
Comment (pratikshaghule) [reply Pratiksha/Javascript/Assignment3/Q3.html]: okk

**Review by VIP5614 — APPROVED** [review]

File: Pratiksha/Javascript/Assignment4/Q3.html (hunk 45 above)
Comment (pratikshaghule) [reply Pratiksha/Javascript/Assignment4/Q3.html:39]: done

**Pull request closed without merging**